app/Console/Commands/Local/MonicaLocalize.php

```diff
@@ -219,4 +221,20 @@ private function fixPagination(array $locales): void
             Storage::disk('lang')->put($locale.DIRECTORY_SEPARATOR.'pagination.php', $pagination);
         }
     }
+
+    private function createLocalNames(array $locales): void
```

Comment:
This is really a detail, but it took me a few seconds to understand what you mean by "localNames".
The true name should be `createLocaleNames` or even `createLocaleFilenames`.

Reply:
good point, I'll fix it